srml/support/procedural/src/storage/impls.rs

```diff
@@ -165,8 +165,8 @@ impl<'a, I: Iterator<Item=syn::Meta>> Impls<'a, I> {
 		} = instance_opts;
 
 		let final_prefix = if let Some(instance) = instance {
-			let method_name = syn::Ident::new(&format!("build_prefix_once_for_{}", name.to_string()), proc_macro2::Span::call_site());
-			quote!{ #instance::#method_name(#prefix.as_bytes()) }
+			let const_name = syn::Ident::new(&format!("PREFIX_FOR_{}", name.to_string()), proc_macro2::Span::call_site());
```

Comment:
Might be good to actually move the prefixes to constants, i.e. `format!("{}{}", PREFIX_FOR, name.to_string())` so that it's not repeated in multiple places

Reply:
yes but at the same time it would be a compile time error